CMakeLists.txt

```diff
@@ -605,9 +605,8 @@ if("${DISTRO_ID}" MATCHES "opensuse" OR
    "${DISTRO_ID}" MATCHES "suse" OR
    "${DISTRO_ID}" MATCHES "sles" )
    set(HCC_GENERAL_RPM_DEP "coreutils, findutils, libelf-devel, pciutils, file, perl-File-Copy-Recursive, perl-File-Listing, perl-File-Which")
-elseif( "${DISTRO_ID}" MATCHES "centos" OR 
-        "${DISTRO_ID}" MATCHES "redhatenterpriseserver" AND 
```

Comment:
I think you should just add the parentheses around the `OR` expression.

Reply:
yeah.  updated.